src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/PublicPrivateDataCombiner.java

```diff
@@ -151,4 +167,61 @@ public Map<String, Pair<CompanyOfficerApi, ManagingOfficerDataApi>> buildMergedM
 
     return combineMO;
   }
+
+  public String logCollatedData() {
+    var result = new StringBuilder("{");
+
+    result.append("\"Managing Officer Data\": [");
+
+    Iterator<Entry<String, Pair<CompanyOfficerApi, ManagingOfficerDataApi>>> moIterator = this.combineMO.entrySet().iterator();
+    while (moIterator.hasNext()) {
+      Entry<String, Pair<CompanyOfficerApi, ManagingOfficerDataApi>> officer = moIterator.next();
+      String[] linkAsArray = officer.getValue().getLeft().getLinks().getSelf().split("/");
+      String publicMoId = linkAsArray[linkAsArray.length - 1];
+
+      result.append("{");
+      result.append("\"Map Key\": \"" + officer.getKey() + "\", ");
```

Comment:
Surely the Map Key == the public hashed id - no need to log both

Reply:
Well it should if it's working correctly I just thought it might be good to log both in case there is an issue somewhere 

I can remove it though